src/Gutenberg/Block_Type.php

```diff
@@ -118,7 +118,7 @@ public function clarkson_render_callback( $attributes, $content ) {
 			);
 		}
 		if ( is_callable( $this->original_render_callback ) ) {
-			return (string) call_user_func( $this->original_render_callback, $attributes, $content );
+			return (string) call_user_func( $this->original_render_callback, $attributes, $content, $block );
```

Comment:
```suggestion
			return (string) call_user_func( $this->original_render_callback, $attributes, $content, $this );
```

I don't think `$block` is available at this point. At least not in the original `block_type` class https://github.com/WordPress/wordpress-develop/blob/6.2/src/wp-includes/class-wp-block-type.php#L398

I'm not sure what to do with this, as WP itself does not seem to be consistent with it. As an example the tags cloud does not seem to have the same parameters: https://github.com/WordPress/wordpress-develop/blob/2beefda2c0448d6dab80d7e7b723058642e57b08/src/wp-includes/blocks/tag-cloud.php#L15

Reply:
When using `$this`, we indeed will always have the proper reference the Clarkson `Block_Type`, so might be better indeed. 
I thought: While there is no Twig template available, we also don't need the Clarkson `Block_Type`, but just use the original `$block` as is. No Clarkson rendering, no Clarkson reference needed.

But ok, maybe, when in the future more functionality might be added to the Clarkson `Block_Type` then `$this` is needed.

In this case, both options work 👍 

About the consistency, when the 3rd argument isn't passed at other places in WordPress Core, it won't matter. It won't trigger an error, right?

Reply:
Just came back to this, because it dawned on me it's a `call_user_func` and not a filter, sorry.

Anyway, using `$this` will be prefered, because `$block` may indeed not be available, but `$this` always is.
These are all, or most of the, render callbacks https://github.com/search?q=repo%3AWordPress%2Fwordpress-develop%20render_block_&type=code

So `call_user_func` is being called, which has a variable-length argument lists as second argument `...`. 
This way, any render callback function with too many arguments will ignore the excess arguments (`$content` or `$block`) without throwing any errors.

I will change it to `$this`.